### ISSUE TITLE
yarn lint: error  Delete `␍`  prettier/prettier

### ISSUE THREAD

**Issue by AmberKiso** [iron-fish/ironfish#923]
### What happened?

`yarn lint` from ironfish root gives a bunch (38k) of beautifying warnings.

### Version

0.1.20

### Relevant log output

```shell
183:37  error  Delete `␍`  prettier/prettier
  184:22  error  Delete `␍`  prettier/prettier
  185:59  error  Delete `␍`  prettier/prettier
  186:6   error  Delete `␍`  prettier/prettier
  187:4   error  Delete `␍`  prettier/prettier
  188:1   error  Delete `␍`  prettier/prettier
  189:44  error  Delete `␍`  prettier/prettier
  190:2   error  Delete `␍`  prettier/prettier

✖ 38006 problems (38006 errors, 0 warnings)
  38006 errors and 0 warnings potentially fixable with the `--fix` option.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run lint --max-warnings=0 stderr:
error Command failed with exit code 1.
lerna ERR! yarn run lint --max-warnings=0 exited 1 in 'ironfish'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by AmberKiso** [comment]
,gitattributes seems to fix this. Trying to set up a PR.

**Comment by AmberKiso** [comment]
Fix proposed in #924

**Comment by deekerno** [comment]
Fixed by #924.

**Comment by AmberKiso** [comment]
Hope this qualifies for testnet credit!
